index.html

```diff
@@ -30,6 +30,7 @@
     <!--<link href="assets/stylesheets/bootstrap.min.css" rel="stylesheet">-->
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.100.2/css/materialize.min.css">
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Quicksand" rel="stylesheet">
```

Comment:
Since this component is getting large, what do you think about splitting this part into a separate component?